tests/testutils/db.py

```diff
@@ -0,0 +1,66 @@
+"""db.py.
+
+Database level utility class for testing against timed_release model.
+"""
+
+from functools import wraps
+import sys
+
+from timed_release import config
+from timed_release.connectors.mysql import base_model
+from timed_release.connectors.mysql import rds_db_engine
+from timed_release.connectors.mysql import rds_db_session
+
+
+def create_all_table():
+    """Create a database table for the respective model classes where.
+
```

Comment:
no its gives error while running flake 8 
either i have to shorten the sentence or let it be as it is